daemon-scheduler/pkg/engine/dispatcher.go

```diff
@@ -82,13 +82,13 @@ func (dispatcher *dispatcher) Start() {
 					}
 				}(event)
 			case <-dispatcher.ctx.Done():
-				log.Infof("Shutting down dispatcher")
+				log.Debug("Shutting down dispatcher")
 				return
 			}
 		}
 	}()
 
-	log.Infof("Started dispatcher")
+	log.Debug("Started dispatcher")
```

Comment:
Should this still be info? Assuming that the dispatcher is a background process, spitting out when it starts may be useful information.